### DISCUSSION TITLE
error:0308010C digital envelope routines unsupported

### DISCUSSION THREAD

**Discussion by SandroFrazaoS** [vercel/next.js#30085] [Help]
hello,
after creating a project using the command
npx create-nextjs-app exercicios

when i execute npm run dev, generates the error
![Screenshot_14](https://user-images.githubusercontent.com/54441280/138006123-456ccb47-cdb2-493b-ba3b-5bef217b6342.png)

**Comment by sethjdev** [comment]
I can reproduce this on node version 17.0.1. I changed my node version to 16.12.0 and reinstalled the dependencies and `yarn dev` works fine now.

**Comment by ijjk** [comment]
Hi, please follow this issue for updates https://github.com/vercel/next.js/issues/30078
